packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -121,56 +121,71 @@ const Services = WebexPlugin.extend({
         this.updateCredentialsConfig();
         catalog.status[serviceGroup].collecting = false;
       })
-      .catch(() => {
+      .catch((error) => {
         catalog.status[serviceGroup].collecting = false;
 
-        return Promise.reject();
+        return Promise.reject(error);
       });
   },
 
   /**
    * Validate if a user, by email, is activated or not. This also returns
-   * an object that contains the atlas user details if available.
+   * an object that contains the atlas user details if available. Currently,
+   * this method has a parameter key for skipping Atlas activation. Once we
+   * have improved our test user package, this will most likely be removed.
    * @param {object} param
    * @param {string} param.email - must be a standard-format email
    * @param {string} [param.reqId] - request id
-   * @returns {Promise<object>} - { activated: boolean, details: string, user: object }
+   * @param {boolean} [param.skipEmail] - skips the atlas request if true
+   * @returns {Promise<object>} output
```

Comment:
Instead of adding lines 141-144, the most jsdoc compatible way would be to do a `@typedef UserValidationObject` above this jsdoc block and then: 
```suggestion
   * @returns {Promise<UserValidationObject>} output
```

Reply:
addressed.